### DISCUSSION TITLE
Is it possible to have some countries rather than the whole list

### DISCUSSION THREAD

**Discussion by jimkaprod** [dr5hn/countries-states-cities-database#705] [Q&A]
Hi there,

as i said in the title is it possible to choose some countries  rather than the whole list.

On the project i'm working on, we know that we only need some countires and we don't want to provide the whole list to the user

Thanks

**Comment by dr5hn** [comment]
@jimkaprod At the present time, you'll need to establish a filter on your end after accessing the API.
